topics/browser/02-browser-apis/00-opening/README.md

```diff
@@ -18,11 +18,6 @@
 Na leitura anterior, descobrimos que HTML5 vem com um conjunto de APIs. Vamos
 entrar um pouco nesse contexto com o seguinte documento:
 
-<iframe
```

Comment:
Cambiar por como el ejemplo anterior.
`[JS Deep Dive](https://docs.google.com/presentation/d/e/2PACX-1vTKtVCzr-absyZJrte790kFPI1QLNLOm1G7QAtSFwexk6nc5ZWFl9H0QY0Io1q-iZO8ArNxAypNdvPm/embed)`

Reply:
This presentation does not exist anymore, that the reason I removed it